gitdash/pages/api/github.js

```diff
@@ -0,0 +1,29 @@
+const { Octokit } = require('@octokit/rest')
```

Comment:
Can you rename this file to github.ts to maintain uniformity?

Reply:
Renamed the file to github.ts